shared_model/builders/protobuf/block.hpp

```diff
@@ -18,105 +18,8 @@
 #ifndef IROHA_PROTO_BLOCK_BUILDER_HPP
 #define IROHA_PROTO_BLOCK_BUILDER_HPP
 
-#include "backend/protobuf/block.hpp"
-#include "backend/protobuf/transaction.hpp"
-#include "block.pb.h"
+#include "builders/protobuf/builder_templates/block_template.hpp"
```

Comment:
What's the purpose of splitting those files? May you explain it a bit?

Reply:
Before we had only `TemplateBlockBuilder` and `BlockBuilder` definition using `using` in the same file. Now we have 2 different builders (`BlockBuilder` and `TestBlockBuilder`) and both of them are dependent from `TemplateBlockBuilder`. IMO putting one of those builder into separate file and another not is not good idea, so it was decided to keep both of them in separate files